src/main/resources/ui/main_window/report.fxml

```diff
@@ -60,6 +60,9 @@
                         <TextArea fx:id="chatLogTextArea" editable="false" prefHeight="200.0" prefWidth="200.0"/>
                     </content>
                 </Tab>
+                <Tab text="Moderator Event">
+                    <TextArea fx:id="moderatorEventTextArea" editable="false" prefHeight="200.0" prefWidth="200.0"/>
```

Comment:
Do we want this to just be a text area or should it be a list box?

Reply:
Text area allows moderators to copy/paste it into their AI assistants 😃

Reply:
yep good point. I always forget about that weakness of javafx